mpas_analysis/shared/plot/plotting.py

```diff
@@ -761,7 +761,10 @@ def plot_vertical_section(
     ylabel=None,
     fileout='moc.png',
     figsize=(10, 4),
-    dpi=300):  # {{{
+    dpi=300,
+    xLim=None,
+    yLim=None,
+    invertYAxis= True):  # {{{
```

Comment:
A reminder for myself: need to make sure this default does not break the plotting of streamfunction MOC (set it to False in the MOC script if it does).